Data/Sys/GameSettings/RODE01.ini

```diff
@@ -25,6 +25,5 @@ PH_ExtraParam = 0
 PH_ZNear =
 PH_ZFar =
 
-[Video_Hacks]
-EFBEmulateFormatChanges = True
-
+[Video_Settings]
+AspectRatio = 2
```

Comment:
Someone might like it better to have stretched widescreen and it will be a pain for a noob to understand what is wrong and how to fix it. As i said it is a matter of preference which is better and it is removed this way.